en/docs/Administer/ProductAdministration/ManagingUsersAndRoles/ManagingUserStores/ConfigurePrimaryUserStore/configuring-a-read-write-active-directory-user-store.md

```diff
@@ -1,190 +1,572 @@
+            GroupNameAttribute="cn"
+            ```
 
     -   Set the `MembershipAttribute` property as shown below:
-
-        ``` html/xml
-                    <Property name="MembershipAttribute">member</Property> 
-        ```
+            ``` 
+            MembershipAttribute="member"
+            ```
 
     To read roles based on a backlink attribute, use the following code snipet instead of the above:
-
-    ``` html/xml
-            <Property name="ReadGroups">false</Property>
-            <Property name="GroupSearchBase">ou=system</Property>
-            <Property name="GroupSearchFilter">(objectClass=groupOfNames)</Property>
-            <Property name="GroupNameAttribute">cn</Property>
-            <Property name="MembershipAttribute">member</Property>
-    <Property name="BackLinksEnabled">true</Property>
-    <Property name="MembershipOfAttribute">memberOf</Property> 
-    ```
+        ```
+        ReadGroups="false"
+        GroupSearchBase="ou=system"
+        GroupNameSearchFilter="(objectClass=groupOfNames)"
+        GroupNameAttribute="cn"
+        MembershipAttribute="member"
+        BackLinksEnabled="true"
+        MembershipOfAttribute="memberOf"
+        ```
+        
 ### Step 2: Updating the system administrator
 
-The **admin** user is the super tenant that will be able to manage all other users, roles and permissions in the system by using the management console of the product. Therefore, the user that should have admin permissions is required to be stored in the user store when you start the system for the first time. Since the Active Directory user store can be written to, you have the option of creating a new admin user in the user store when you start the system for the first time. Alternatively, you can also use a user ID that already exists in the user store. For more information on setting up the system administrator and the authorization manager, see [Configuring the User Realm](https://docs.wso2.com/display/ADMIN44x/Configuring+the+User+Realm) .
+The **admin** user is the super tenant that will be able to manage all other users, roles and permissions in the system by using the management console of the product. Therefore, the user that should have admin permissions is required to be stored in the user store when you start the system for the first time. Since the Active Directory user store can be written to, you have the option of creating a new admin user in the user store when you start the system for the first time. Alternatively, you can also use a user ID that already exists in the user store. For more information on setting up the [system administrator](https://apim.docs.wso2.com/en/latest/Administer/ProductAdministration/ManagingUsersAndRoles/ManagingUserStores/configuring-the-system-administrator/) and the [authorization manager](https://apim.docs.wso2.com/en/latest/Administer/ProductAdministration/ManagingUsersAndRoles/ManagingUserStores/configuring-the-authorization-manager/) .
```

Comment:
Please use {{basepath}} here, and wherever applicable

Reply:
Fixed with c0c321d